pages/1.10/monitoring/logging/quickstart/index.md

```diff
@@ -48,7 +48,9 @@ Deploy a sample Marathon app for use in this quick start guide.
     test-log  10.0.1.105  root    R    test-log.e69c4b2f-c255-11e6-a451-aa711cbcaa78
     ```
 
-# View the Mesos and DC/OS logs
+# View the DC/OS (Mesos) task logs
+
+You can access the Mesos stderr and stdout logs natively through the DC/OS CLI `dcos task log` command. In this example, a task is launched and the stderr and stdout logs from Mesos are accessed.
 
 You can access the Mesos stderr and stdout logs natively through the DC/OS CLI `dcos task log` command. In this example, a task is launched and the stderr and stdout logs from Mesos are accessed.
```

Comment:
I would add the word "task" here: 
> You can access the Mesos task stderr and stdout logs natively...

I would delete the following sentence as It's confusing and doesn't give additional information:

> In this example, a task is launched and the stderr and stdout logs from Mesos are accessed.

Reply:
Fixed.